sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -258,21 +285,23 @@ async def create_database(
         offer_throughput: Optional[Union[int, ThroughputProperties]] = None,
         initial_headers: Optional[Dict[str, str]] = None,
         throughput_bucket: Optional[int] = None,
+        return_headers: Optional[bool] = False,
         **kwargs: Any
-    ) -> DatabaseProxy:
+    ) -> Union[DatabaseProxy, CosmosDict]:
         """
         Create a new database with the given ID (name).
 
         :param str id: ID (name) of the database to create.
         :keyword offer_throughput: The provisioned throughput for this offer.
         :paramtype offer_throughput: Union[int, ~azure.cosmos.ThroughputProperties]
+        :param return_headers: Specifies function to return either a DatabaserProxy or a CosmosDict instance.
         :keyword dict[str, str] initial_headers: Initial headers to be sent as part of the request.
         :keyword response_hook: A callable invoked with the response metadata.
         :keyword int throughput_bucket: The desired throughput bucket for the client
         :paramtype response_hook: Callable[[Dict[str, str], Dict[str, Any]], None]
         :raises ~azure.cosmos.exceptions.CosmosResourceExistsError: Database with the given ID already exists.
-        :returns: A DatabaseProxy instance representing the new database.
-        :rtype: ~azure.cosmos.aio.DatabaseProxy
+        :returns: A DatabaseProxy instance representing the database or a CosmosDict with the response headers
```

Comment:
```suggestion
        :returns: A DatabaseProxy instance representing the database or the database properties.
```
I wouldn't call out `CosmosDict` by name - in Python terms it's just a dict that happens to have an extra method. Instead, I would describe what this dict represents - in this case it's the properties of the newly created database.